lib/server.js

```diff
@@ -15,11 +15,11 @@ class ServerBase {
     this.pendingReady = new Set()
   }
 
-  static serveIndexPage (res, torrents) {
+  static serveIndexPage (res, torrents, pathname) {
     const listHtml = torrents
       .map(torrent => (
       `<li>
-        <a href="${escapeHtml(torrent.infoHash)}">
+        <a href="${escapeHtml(pathname)}/${escapeHtml(torrent.infoHash)}">
```

Comment:
infoHash is always hex, it doesn't need to be escaped

Reply:
I only used escapeHtml as it appeared to be used everywhere in the serveIndex functions. escapeHtml was already being used on [`file.length`](https://github.com/webtorrent/webtorrent/blob/4d0267600b405aad40759e41f1b35130972ba880/lib/server.js#L82), so I assumed the intention was to use it everywhere

Reply:
yeah would be better, it was there before which was kinda my bad, I had a massive workload when I was rewriting all the http server functionality, and it's still gonna be re-written again